update.sh

```diff
@@ -158,5 +158,10 @@ wait
 
 post_to_slack $MY_EXIT_CODE
 
+if [ -x "/app/shutdown.sh" ]; then 	
```

Comment:
good idea on how to include it optionally default off to not make waves

Reply:
yeah, i mean, it will be part of the image but i never assume things :)